src/libtriton/includes/triton/astContext.hpp

```diff
@@ -210,6 +210,15 @@ namespace triton {
             }
           }
 
+          if (this->modes->isModeEnabled(triton::modes::AST_OPTIMIZATIONS) &&
+              node->getBitvectorSize() <= 512) {
```

Comment:
&& node->getBitvectorSize() <= 512 is always true?

Reply:
No longer need this check. I will remove it.

Reply:
Done